app/scripts/lib/mailcheck.js

```diff
@@ -30,8 +30,9 @@ function (Tooltip) {
    * @param {Object} target DOM input node to target with mailcheck
    * @param {Object} metrics Metrics logger
    * @param {Object} translator Translator object passed by the view
+   * @param {Object} experiments Experiments object passed by the view
    */
-  return function checkMailInput(target, metrics, translator) {
+  return function checkMailInput(target, metrics, translator, experiments) {
```

Comment:
@shane-tomlinson rolled back to old mailcheck setup, with an additional "experiments" option. Still WIP but you can see now that I need to make sure I can save the state inside of mailcheck. However as you were saying about "promoting features from experiments" should be easier. if we want to promote mailcheck we just remove the experiment option and done.